app/src/main/res/layout/card_meditation.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/meditation_card"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:cardCornerRadius="12.0dip"
+    app:cardElevation="8.0dip"
+    app:cardPreventCornerOverlap="false"
+    app:cardUseCompatPadding="true">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/gradient_meditation">
+
+        <ImageView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:adjustViewBounds="true"
+            android:alpha="0.3"
+            android:scaleType="fitCenter"
+            android:tint="#fff"
+            app:srcCompat="@drawable/bg_card_meditation" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:orientation="horizontal"
+            android:padding="16.0dip">
+
+            <ImageView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginStart="8dp"
+                android:layout_marginEnd="8dp"
+                android:layout_weight="1.9"
+                android:src="@drawable/ic_card_meditation"
+                android:tint="#fff" />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:orientation="vertical">
+
+                <android.support.v7.widget.AppCompatTextView
+                    android:id="@+id/meditation_title"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:paddingStart="12.0dip"
+                    android:paddingTop="12.0dip"
+                    android:paddingEnd="12.0dip"
+                    android:paddingBottom="0dip"
+                    android:text="Meditation"
+                    android:textAppearance="@style/TextAppearance.AppCompat.Display1"
+                    android:textColor="#fff" />
+
+                <android.support.v7.widget.AppCompatTextView
+                    android:id="@+id/meditation_subtitle"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:padding="12.0dip"
+                    android:text="to (cause someone to) become less active and more calm and happy"
```

Comment:
@adityastic The text labels and descriptions are already present in `strings.xml`. Please use from there,

Reply:
great